mantle/kola/harness.go

```diff
@@ -761,6 +761,16 @@ func registerTestDir(dir, testprefix string, children []os.FileInfo) error {
 			if err := registerTestDir(subdir, subprefix, subchildren); err != nil {
 				return err
 			}
+		} else if isreg && (c.Mode().Perm()&0001) == 0 {
+			file, err := os.Open(filepath.Join(dir, c.Name()))
+			if err != nil {
+				return errors.Wrapf(err, "opening %s", c.Name())
+			}
+			scanner := bufio.NewScanner(file)
+			scanner.Scan()
+			if scanner.Text() == "#!/bin/bash" {
```

Comment:
```suggestion
			if strings.HasPrefix(scanner.Text(), "#!") {
```

will cover also e.g. `/usr/bin/bash` as well as non-shell interpreters (`#!/usr/bin/env python` etc.).

Reply:
Right, this is much better